### PR TITLE
Buffs eye contact, adds "Shifty Eyes" quirk

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
So back in #52108 I added eye contact, where looking at someone within one second of them looking at you (assuming you were within 5 tiles and didn't have hidden faces) would give you both a little message about making eye contact. I think it's a really little neat feature, but I basically never see it proc because of how short the time window is for it.

This PR extends the time window for making eye contact to 2 seconds, as well as adding a new trait called "Shifty Eyes" that rarely (10% - 1% for each tile of distance between the person and you) makes someone who looks at you think you both made eye contact. 

Codewise, this PR decouples examine_more and eye contact sharing the same time window. Atoms are removed from recent_examines after 2 second rather than 1 though behavior-wise, examine_more still works the same, examining multiple times in one second still only makes one check for eye contact.

## Why It's Good For The Game
The few times I've seen this happen in game or heard players talk about having it happen, they were caught off guard and thought it was cool in an immersion/paranoia sense. The numbers here might need some tweaking to find a good balance, but I think a moderate buff to how often it happens would be positive.

## Changelog

:cl: Ryll/Shaps
add: Added new quirk, "Shifty Eyes", which rarely causes people who examine you while standing closeby to think that you both made eye contact, even when you didn't.
balance: The time window for making eye contact with someone else has been extended from  examining each other within 1 second to within 2 seconds.
/:cl:

### CODE REVIEW

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -547,9 +546,17 @@
...
 
 /mob/living/handle_eye_contact(mob/living/examined_mob)
-	if(!istype(examined_mob) || src == examined_mob || examined_mob.stat >= UNCONSCIOUS || !client || !examined_mob.client?.recent_examines || !(src in examined_mob.client.recent_examines))
+	if(!istype(examined_mob) || src == examined_mob || examined_mob.stat >= UNCONSCIOUS || !client)
 		return
 
+	var/imagined_eye_contact = FALSE
+	if(!LAZYACCESS(examined_mob.client?.recent_examines, src))
+		// even if you haven't looked at them recently, if you have the shift eyes trait, they may still imagine the eye contact
+		if(HAS_TRAIT(examined_mob, TRAIT_SHIFTY_EYES) && prob (10 - get_dist(src, examined_mob)))
```
Comment (Mothblocks) [inline code/modules/mob/mob.dm:555]:
```suggestion
		if(HAS_TRAIT(examined_mob, TRAIT_SHIFTY_EYES) && prob(10 - get_dist(src, examined_mob)))
```

File: code/datums/quirks/neutral.dm [hunk 2]
```diff
@@ -231,6 +231,14 @@
 
 	give_item_to_holder(wayfinder, list(LOCATION_LPOCKET = ITEM_SLOT_LPOCKET, LOCATION_RPOCKET = ITEM_SLOT_RPOCKET, LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
 
+/datum/quirk/shifty_eyes
+	name = "Shifty Eyes"
+	desc = "Your eyes tend to wander all over the place, whether you mean to or not, causing people to sometimes think you're looking directly at them when you aren't."
+	icon = "far fa-eye"
```
Comment (Mothblocks) [inline code/datums/quirks/neutral.dm:237]: What does this look like? I forget what the far discriminant is.

File: code/__DEFINES/traits.dm [hunk 3]
```diff
@@ -497,6 +497,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_INTROVERT "introvert"
 #define TRAIT_ANXIOUS "anxious"
 #define TRAIT_INSANITY "insanity"
+#define TRAIT_SHIFTY_EYES "shifty_eyes"
```
Comment (Mothblocks) [inline code/__DEFINES/traits.dm:500]: Document

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/datums/quirks/neutral.dm (hunk 2 above)
Comment (Ryll-Ryll) [reply code/datums/quirks/neutral.dm:237]: tbh i got no clue, i just copied what the fontawesome site told me. Should I just try `eye`?

**Comment by Ryll-Ryll** [comment]
I'd like to get this TM'd for a bit to make sure it's not going to become way too common as well

File: code/datums/quirks/neutral.dm (hunk 2 above)
Comment (Mothblocks) [reply code/datums/quirks/neutral.dm:237]: They'll be different images, try both and see which one you like better, then post a picture of it alongside the other quirk icons.

**Comment by EricZilla** [comment]
Shifty-Eyes sounds like the quirk you'd take to make people NOT notice when you are looking at them to me, dunno why. Maybe Wandering Eyes would be more clear? Obviously not a very big deal

**Comment by Mothblocks** [comment]
daydreamer

**Pull request merged**